flink-core/src/main/java/org/apache/flink/configuration/MetricOptions.java

```diff
@@ -135,14 +139,18 @@
 	 */
 	public static final ConfigOption<Boolean> SYSTEM_RESOURCE_METRICS =
 		key("metrics.system-resource")
-			.defaultValue(false);
+			.defaultValue(false)
+			.withDescription("Flag indicating whether Flink should report system resource metrics such as machine's CPU," +
+				" memory or network usage.");
 	/**
 	 * Interval between probing of system resource metrics specified in milliseconds. Has an effect only when
 	 * {@link #SYSTEM_RESOURCE_METRICS} is enabled.
 	 */
 	public static final ConfigOption<Long> SYSTEM_RESOURCE_METRICS_PROBING_INTERVAL =
 		key("metrics.system-resource-probing-interval")
-			.defaultValue(5000L);
+			.defaultValue(5000L)
+			.withDescription("Interval between probing of system resource metrics specified in milliseconds. Has an effect" +
+				" only when SYSTEM_RESOURCE_METRICS is enabled");
```

Comment:
please refer to the actual configuration key for `SYSTEM_RESOURCE_METRICS ` instead, i.e. `"Interval ... only when '" + SYSTEM_RESOURCE_METRICS.key() + "' is enabled."`.

The description is also missing a final period.